api/lib/validation/attributeValidator.js

```diff
@@ -76,36 +83,81 @@ var validateRecurringScheduleAttributeValue = function(inputRecurringSchedules)
     if(inputRecurringSchedules[i].instance_min_count >= 
       inputRecurringSchedules[i].instance_max_count) {
       let error = createErrorResponse('recurring_schedule.instance_min_count',
-      'recurring_schedule.instance_min_count and recurring_schedule.instance_max_count ' + 
-      'values are not compatible',inputRecurringSchedules[i],'instance_min_count ' + 
-       inputRecurringSchedules[i].instance_min_count + ' is higher or equal to ' + 
-       'instance_max_count ' + inputRecurringSchedules[i].instance_max_count +
-       ' in recurring_schedule :[' + i + ']');
+          'recurring_schedule.instance_min_count and recurring_schedule.instance_max_count ' + 
+          'values are not compatible',inputRecurringSchedules[i],'instance_min_count ' + 
+          inputRecurringSchedules[i].instance_min_count + ' is higher or equal to ' + 
+          'instance_max_count ' + inputRecurringSchedules[i].instance_max_count +
+          ' in recurring_schedule :[' + i + ']');
       attributeValueErrors[errorCount++] = error;
     }
     if(!compareTimes(inputRecurringSchedules[i].start_time,
```

Comment:
How? Your compareDates does a strict isAfter. If startDate == endDate, you fail.